lib/dev-requirements.txt

```diff
@@ -24,3 +24,4 @@ types-requests
 types-setuptools
 types-toml
 types-typed-ast
+pandas-stubs
```

Comment:
`pandas-stubs` would be a good idea to add 👍  But this will need a bit more time and a dedicated PR since it is currently running into many typing issues. Seems to be mainly in tests, but also some within the codebase. We can take a look into this next quarter during the stability efforts.